### PR TITLE
Feat: Multisend wrapper

### PR DESCRIPTION
**Pull request by cbrzn**

### CODE REVIEW

File: packages/apis/multisend/scripts/createPayload.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { ethers } = require("hardhat");
+const fs = require("fs");
```
Comment (Niraj-Kamdar) [inline packages/apis/multisend/scripts/createPayload.js:2]: What's the use of these scripts? If it's for just testing then we should create jest tests instead

File: packages/apis/multisend/deploy/00_delegator.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module.exports = async ({ deployments, getNamedAccounts }) => {
+  const { deployer } = await getNamedAccounts();
+  const { deploy } = deployments;
```
Comment (Niraj-Kamdar) [inline packages/apis/multisend/deploy/00_delegator.js:3]: Let's use typescript here.

**Review by Niraj-Kamdar — CHANGES_REQUESTED** [review]
We should move this to integrations repo.

**Comment by cbrzn** [comment]
> We should move this to integrations repo.

Hey @Niraj-Kamdar I totally agree with you - Currently, this wrapper is not working as expected because it only works with Contract Accounts and not with EOAs (because of the `msg.sender`). I will update this and use instead the multicall library (In the integrations repository)

**Comment by dOrgJelli** [comment]
Closing this for now @cbrzn

**Pull request closed without merging**